### ISSUE TITLE
ban immenent

### ISSUE THREAD

**Issue by ArthurPetrilli** [PokeBotPub/PokeBot-Android#334]
![2017 03 09 warning](https://cloud.githubusercontent.com/assets/24369857/23778057/48168276-0506-11e7-9c86-89e72659ff65.png)

**Comment by ArthurPetrilli** [comment]
Hey Developers, This feels like January ban wave all over again.  Everyone needs to stop using the bot until Developers fix this.  I may have 2 accts on the chopping block soon.....i stopped all other accts....did not get the msg....but i bet it will be a matter of time.....that is NIantic's MO.....This is getting real old....i am that much closer to switching to Pokemon Sun

**Comment by ShairaMaex** [comment]
i survive. past ban waves and the January BanWave.

**Comment by ArthurPetrilli** [comment]
When i lose accts, i make new one.  Making new acct is getting old.  This happen with my young accts - slow and steady is not doing shit....Niantic is somehow detecting the pokebot.  It is a good thing i did not use pokebot with my higher level accts.

**Comment by ShairaMaex** [comment]
well Slow and stead works for me. just fimish maxing 4 of my accs i get
bakely fresh acc is at 28

**Comment by Thesse57** [comment]
For me his walking speed on 5 everything is fine

**Comment by PokeBotPub** [comment]
Pokebot 1.0.41 update is for this problem.

**Comment by ArthurPetrilli** [comment]
Do i need to wait until my acct is free of niantic security checks b4 i run program? Last time, December,  i had to wait a week b4 the security check was gone

**Comment by AcheronKaa** [comment]
Hi ShairaMaex, I was hoping to ask you if you could share your pokebot settings with me please. I'm not that good at figuring out the settings or where I should not and I really want to level at least one account. If your could reply @ raugust1982@gmail.com I would greatly appreciate. Thank you

**Comment by ShairaMaex** [comment]
of course, theb i separate my bot and pogo all. with different devices

**Comment by ArthurPetrilli** [comment]
Today, I have updated to the latest bot 1.0.42.  i used the program on an acct that i have not botted with since 2 weeks ago.  This very acct was used to log in yesterday to verify if i had a ban warning on it....it did not have a ban warning.   After 1 hour of run time using the latest bot 1.0.42 i gave the bot a dinner break.  .  I went to dinner and eventually logged in to the regular pogo.  Ban Warning came up!!!!  All my 2nd string acct from yesterday now have a ban warning!!!!  Today, all my 1st string accts have a ban warning after running latest bot 1.0.42 for 2 session at 60 minutes each.  I have created 2 3rd string accts to see how high i can get and if i get ban warnings on them.....I feel like all my accts are at risk of next ban wave (imminent) Cold storage for about 2 weeks...maybe a month.  Time for 3rd string to build up and test the security threats!!!!!
